.env.example

```
@@ -1,14 +1,99 @@
+# --- Authentication ---
+
+# Alpaca -> issuance (min 32 chars); used on ITN endpoints Alpaca calls
 ISSUER_API_KEY=${ISSUER_API_KEY}
+# Comma-separated CIDRs; empty disables IP filtering (not for production)
 ALPACA_IP_RANGES=${ALPACA_IP_RANGES}
 INTERNAL_IP_RANGES=127.0.0.0/8,::1/128,172.16.0.0/12,${INTERNAL_IP_RANGES_EXTRA}
+
+# --- Alpaca broker API (issuance -> Alpaca) ---
+
 ALPACA_API_BASE_URL=${ALPACA_API_BASE_URL}
 ALPACA_ACCOUNT_ID=${ALPACA_ACCOUNT_ID}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALPACA_ACCOUNT_ID key should go before the ALPACA_API_BASE_URL key

(UnorderedKey)